src/Tizen.NUI/src/internal/WebView.cs

```diff
@@ -871,5 +879,19 @@ public void SetTtsFocus(bool focused)
             Interop.WebView.SetTtsFocus(SwigCPtr, focused);
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
+
+        /// <summary>
+        /// Get a plain text of current web page asynchronously.
+        /// Please note that it gets plain text of currently loaded page so please call this method after page load finished.
+        /// <param name="callback">The callback for getting plain text</param>
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void GetPlainTextAsynchronously(PlainTextReceivedCallback callback)
```

Comment:
Use Task
```suggestion
        public Task<string> GetPlainTextAsync()
```

Reply:
Since it is cherry-picking patch from the API9,
so I will check to apply `Task` to the API9.